tests/cluster_tests/torque_custom/torque_mode.py

```diff
@@ -19,179 +19,111 @@
 import Simulation
 import math
 
-def createAndRunQSUB(simulation):
+def createAndRunQSUB(runInfoDict):
   """
     Generates a PBS qsub command to run the simulation
-    @ In, simulation, instance, instance of the simulation class
-    @ Out, None
+    @ In, runInfoDict, dict, dictionary of run info.
+    @ Out, remoteRunCommand, dict, dictionary of command.
   """
   # Check if the simulation has been run in PBS mode and, in case, construct the proper command
   #while true, this is not the number that we want to select
-  coresNeeded = simulation.runInfoDict['batchSize']*simulation.runInfoDict['NumMPI']
-  #batchSize = simulation.runInfoDict['batchSize']
-  frameworkDir = simulation.runInfoDict["FrameworkDir"]
-  ncpus = simulation.runInfoDict['NumThreads']
-  jobName = simulation.runInfoDict['JobName'] if 'JobName' in simulation.runInfoDict.keys() else 'raven_qsub'
+  coresNeeded = runInfoDict['batchSize']*runInfoDict['NumMPI']
+  #batchSize = runInfoDict['batchSize']
+  frameworkDir = runInfoDict["FrameworkDir"]
+  ncpus = runInfoDict['NumThreads']
+  jobName = runInfoDict['JobName'] if 'JobName' in runInfoDict.keys() else 'raven_qsub'
   #check invalid characters
   validChars = set(string.ascii_letters).union(set(string.digits)).union(set('-_'))
   if any(char not in validChars for char in jobName):
-    simulation.raiseAnError(IOError,'JobName can only contain alphanumeric and "_", "-" characters! Received'+jobName)
+    raise IOError('JobName can only contain alphanumeric and "_", "-" characters! Received'+jobName)
```

Comment:
As I commented elsewhere, these modes are not part of the raven core and so there is no expectation that they should handle printing and error raising through the messageHandler.